README.md

```diff
@@ -98,8 +98,10 @@ sha3-384,           ,                         0x15
 sha3-512,           ,                         0x14
 shake-128,          ,                         0x18
 shake-256,          ,                         0x19
+murmur3             ,                         0x22
 blake2b,            ,                         0x40
 blake2s,            ,                         0x41
+dbl-sha2-256        ,                         0x56
```

Comment:
Have you considered if we can have a way to make this more generic? Reserving a code for a multiple of a hash function seems odd (it is an easy way to throw a lot of codes.

I'm was wondering if we should make this broader and have byte in multihash (specifically multihash) that says how many times it got used, but then again, that would break all the current hashes :( So maybe we have to make this exceptions afterall